src/app/home/home.component.html

```diff
@@ -102,11 +103,11 @@ <h1>contact me</h1>
 <footer>
   <p>Follow me on social media</p>
   <div class="social-icons">
-    <img class="social-icon" src="img/github.svg" alt="github">
-    <img class="social-icon" src="img/linkedin.svg" alt="linkedin">
-    <img class="social-icon" src="img/instagram.svg" alt="instagram">
+    <img class="social-icon" src="assets/img/github.svg" alt="github">
+    <img class="social-icon" src="assets/img/linkedin.svg" alt="linkedin">
+    <img class="social-icon" src="assets/img/instagram.svg" alt="instagram">
 
   </div>
 
-  <p class="copyright">Copywrite 2024 Alex Nesteruk</p>
+  <p class="copyright">Copyright 2024 Alex Nesteruk</p>
```

Comment:
The copyright year should be a function to get the current year. This way you do not need to update it every year.